src/imitation/policies/base.py

```diff
@@ -65,3 +69,27 @@ class FeedForward32Policy(policies.ActorCriticPolicy):
     def __init__(self, *args, **kwargs):
         """Builds FeedForward32Policy; arguments passed to `ActorCriticPolicy`."""
         super().__init__(*args, **kwargs, net_arch=[32, 32])
+
+
+class NormalizeFeaturesExtractor(torch_layers.FlattenExtractor):
```

Comment:
It will. I don't think we want to normalize CNN input anyway (at least adaptively), as images are usually in a fixed range? I could try to auto-detect whether we're using an MLP or CNN, but I'm reluctant to add too much config magic here.

Reply:
Not sure how to solve this so leaving for now, but happy to address it in another PR if you have suggestions.

Reply:
No, that's totally reasonable—I don't normally images most of the time either.